test/components.js

```diff
@@ -13,7 +14,7 @@ test.each(
     .filter(key => !blacklist[key])
     .map(key => {
       const Component = components[key]
-      return [ key, <Component /> ]
+      return [ key, <ThemeProvider theme={{}}><Component /></ThemeProvider> ]
```

Comment:
Yeah, the other thing to do for components that rely on the theme is to provide it as a default prop, but this works as well